src/test/java/teammates/test/cases/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -47,6 +53,23 @@ private void testModerationHint() throws Exception {
         assertEquals("[More]", submitPage.getModerationHintButtonText());
     }
 
+    private void testAddCommentsToQuestionsWithoutResponse() throws IOException {
+        ______TS("add comments on questions without responses: no effect");
+
+        logout();
+        submitPage = loginToInstructorEditStudentFeedbackPage(
+                "IESFPTCourseinstr", "student1InIESFPTCourse@gmail.tmt", "session1InIESFPTCourse");
+        submitPage.waitForPageToLoad();
+
+        submitPage.addFeedbackResponseComment("-3-0", "Comment without response");
```

Comment:
Need create a issue regarding this.

We shall hidden the `add comment` button when there is no comment. No need to do it in this PR if needs huge efforts.

Reply:
Okay I'll create issue for it.